docs/docs/legal/terms-support-service.md

```diff
@@ -0,0 +1,222 @@
+
+### Support Process
+
+You will ensure to follow the support process, especially provide all required initial information to the issue, as outlined in the [Annex](support-services) to this document.
+
+### Training of support staff
+
+You will ensure regular training of your support staff. Your support staff must be able to provide the required information for support issues to us, and thus requires access and up-to-date knowledge of the services.
+
+Initial know-how transfer for the services will be organized in training sessions conducted by us. We can provide knowledge sessions throughout the term to train newly onboarded staff, update your support staff about important updates, or refresh knowledge in specified areas. In case we notice insufficient quality of support requests from Customers, we will propose appropriate training sessions.
+
+### Security
+
+You will take appropriate measures to prevent any misuse of the services you booked. These include, for example, securing the software used and the prompt installation of security updates as well as using suitably secure passwords.
+
+### Disaster recovery
+
+We take care of the necessary disaster recovery measures. The goal is to maintain a maximum 24h old restore point off all the vital data.
+
+Any liability for damages, indirect or direct, in case of data loss is explicitly rejected.
+
+### Reporting obligations
+
+You will immediately report any knowledge of a misuse of your booked services.
+
+### Cooperation
+
+If the maintenance of service quality requires your cooperation, for example to remedy errors in the services you use, you will provide said cooperation promptly and free of charge.
+
+### Third party obligations
+
+You will ensure that your vicarious agents, customers and third parties fulfill these obligations as well.
+
+## Financial
+
+### Payment
+
+Fees for the initial or any subsequent term of Support Services booked will be due and payable net 30 days from date of invoice. All payments to CAOS will be made in CHF or EUR to our bank account, as indicated in the PO.
+
+### Offsetting
+
+Offsetting against a counterclaim is prohibited.
+
+### Collection
+
+In the event of default we reserve the right to transfer our claim to a collections agency. You will bear any resulting costs insofar as legally permissible.
+
+### Lapsed Service Fee
+
+In case the term of the Support Service contract has expired within 1 to 180 days, you will be required to pay a Lapsed Service fee in addition to purchasing and activating a one-year renewal contract term at the then-current fee and conditions. The renewal term's start date will also be backdated to begin coverage from the service's original expiration date.
+
+Please contact us for current fees.
```

Comment:
```suggestion
Please contact us for the current fees.
```

Reply:
'the' is not necessary here, as we speak of the general fees. We could say 'contact us for the current lapsed service fee'.